tests/app/clients/test_aws_ses.py

```diff
@@ -80,6 +82,111 @@ def test_send_email_handles_reply_to_address(notify_api, mocker, reply_to_addres
         assert f"reply-to: {expected_value}" in raw_message
 
 
+def test_send_email_txt_and_html_email(notify_api, mocker):
+    boto_mock = mocker.patch.object(aws_ses_client, '_client', create=True)
+    mocker.patch.object(aws_ses_client, 'statsd_client', create=True)
+
+    with notify_api.app_context():
+        aws_ses_client.send_email(
+            'from@example.com',
+            to_addresses='destination@example.com',
+            subject='Subject',
+            body='email body',
+            html_body='<p>email body</p>',
+            reply_to_address='reply@example.com',
+        )
+
+    boto_mock.send_raw_email.assert_called_once()
+    raw_message = boto_mock.send_raw_email.call_args.kwargs['RawMessage']['Data']
+
+    regex = dedent(r"""
+        Content-Type: multipart\/alternative; boundary="===============(?P<boundary>.+)=="
```

Comment:
is there a reason why we are capturing name groups?

Reply:
Yes, those name groups are here to prove that envelopes have been set as expected. Some things need to be contained within things and groups are boundaries https://github.com/cds-snc/notification-api/blob/9ca52c3c0d44f745b4097b9a6d72cf854b6e3431/app/clients/email/aws_ses.py#L95-L106